Stylesheets/Components/badge.css

```diff
@@ -0,0 +1,81 @@
+.badge{
+    position: relative;
+    margin:1rem;
+ 
```

Comment:
use prettier to format your code, don't leave these extra spaces

Reply:
Thanks vanshita will use prettier before pushing the code.